Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -27,25 +39,63 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
   CommentRepliesViewModelInputs,
   CommentRepliesViewModelOutputs {
   public init() {
-    let rootComment = Signal.combineLatest(
-      self.commentProperty.signal.skipNil(),
+    let rootCommentProject = Signal.combineLatest(
+      self.commentProjectProperty.signal.skipNil(),
       self.viewDidLoadProperty.signal
     )
     .map(first)
 
+    let rootComment = rootCommentProject.map(first)
+
+    let project = rootCommentProject.map(second)
+
     self.loadCommentIntoDataSource = rootComment
+
+    let currentUser = self.viewDidLoadProperty.signal
+      .map { _ in AppEnvironment.current.currentUser }
+
+    let inputAreaBecomeFirstResponder = rootCommentProject
+      .map(third)
+      .takeWhen(self.viewDidAppearProperty.signal)
+
+    self.configureCommentComposerViewWithData = Signal
+      .combineLatest(
+        project,
+        currentUser.signal,
+        self.viewDidLoadProperty.signal,
+        inputAreaBecomeFirstResponder
+      )
+      .map { ($0.0, $0.1, $0.3) }
+      .map { project, currentUser, inputAreaBecomeFirstResponder in
+        let isBacker = userIsBackingProject(project)
+        let isCreatorOrCollaborator = !project.memberData.permissions.isEmpty && !isBacker
```

Comment:
This line specifically will can also to be added to the `let isNotABacker` signal in `CommentCellViewModel` because that logic doesn't not cover `project.memberData.permissions`. It does however control the reply button show/hide, so if the user is a creator/collaborater they should be able to see the reply button. Just a small miss along the way.

Reply:
This is an issue currently logged [NT-2034](https://kickstarter.atlassian.net/browse/NT-2034)

Reply:
I'm going to skip it for this PR.

Reply:
Ok, if you can make a note in the ticket to reference this comment on GH, that would help when we return to it.